packages/kbn-journeys/journey/journey_ftr_config.ts

```diff
@@ -83,11 +83,14 @@ export function makeFtrConfigProvider(
         ...baseConfig.kbnTestServer,
         // delay shutdown by 15 seconds to ensure that APM can report the data it collects during test execution
         delayShutdown: process.env.TEST_PERFORMANCE_PHASE === 'TEST' ? 15_000 : 0,
+        // delay shutdown by 1 second to allow the Kibana server to complete the flush of the events in the queue
+        delayEndOfTest: process.env.TEST_PERFORMANCE_PHASE === 'TEST' ? 1_000 : 0,
```

Comment:
adding a delay after calling `stop` of the server. My theory is that we kill the node process right after calling `stop`. IMO, 1s should be enough to push the enqueued telemetry (unless there's a high event-loop delay due to the load, which I don't think it's the case).

Reply:
I don't think we need this anymore since we _fixed_ the server-side :)